app/controllers/ontologies_controller.rb

```diff
@@ -437,6 +437,67 @@ def metrics_evolution
     render partial: 'ontologies/sections/metadata/metrics_evolution_graph', locals: { data: data }
   end
 
+  def selector
+    @categories = LinkedData::Client::Models::Category.all(display_links: false, display_context: false)
+    @groups = LinkedData::Client::Models::Group.all(display_links: false, display_context: false)
+    @filters = ontology_filters_init(@categories, @groups)
+    @select_id = params[:id]
+    render 'ontologies/selector/selector' , layout: false
+  end
+
+  def selector_results
+    api_params = {}
```

Comment:
regroup the usage of `params` only in this line; dispatching it everywhere makes it hard to know what is the parameter of this endpoint